spec.html

```diff
@@ -11723,9 +11723,8 @@ <h1>
 
   <emu-clause id="sec-execution-contexts">
     <h1>Execution Contexts</h1>
-    <p>An <dfn variants="execution contexts">execution context</dfn> is a specification device that is used to track the runtime evaluation of code by an ECMAScript implementation. At any point in time, there is at most one execution context per agent that is actually executing code. This is known as the agent's <dfn id="running-execution-context" variants="running execution contexts">running execution context</dfn>. All references to the running execution context in this specification denote the running execution context of the surrounding agent.</p>
-    <p>The <dfn id="execution-context-stack" variants="execution context stacks">execution context stack</dfn> is used to track execution contexts. The running execution context is always the top element of this stack. A new execution context is created whenever control is transferred from the executable code associated with the currently running execution context to executable code that is not associated with that execution context. The newly created execution context is pushed onto the stack and becomes the running execution context.</p>
-    <p>An execution context contains whatever implementation specific state is necessary to track the execution progress of its associated code. Each execution context has at least the state components listed in <emu-xref href="#table-state-components-for-all-execution-contexts"></emu-xref>.</p>
+    <p>An <dfn variants="execution contexts">execution context</dfn> brings together information that is relevant for the runtime evaluation of some unit of code (e.g., a script, a module, or a function object). At any point in time, there is at most one execution context per agent whose associated code is being evaluated. This is known as the agent's <dfn id="running-execution-context" variants="running execution contexts">running execution context</dfn>. All references to the running execution context in this specification denote the running execution context of the surrounding agent.</p>
```

Comment:
```suggestion
    <p>An <dfn variants="execution contexts">execution context</dfn> brings together information that is relevant for the runtime evaluation of some unit of code (e.g., a script, a module, or a function object). At any point in time, there is at most one execution context per agent whose associated code is being evaluated. This is known as the agent's <dfn id="running-execution-context" variants="currently running execution context">running execution context</dfn>. All references to the running execution context in this specification denote the running execution context of the surrounding agent.</p>
```

1. it should never have to be plural since there can only be one
2. we say "currently" before it below, and I'd rather that just be included in an alias for this term

Reply:
> 1. it should never have to be plural since there can only be one

Only one per agent, but if you were taking about multiple agents, then you could conceivably refer to their running execution contexts. That seems unlikely though, so I've removed the plural variant.

>     2. we say "currently" before it below, and I'd rather that just be included in an alias for this term

Hm. To me, "the currently running execution context" reads like "the execution context that is currently running", which goes against one of the ideas of this PR (execution contexts don't 'run'), so I've changed all such phrases by deleting "currently".